Cargo.toml

```diff
@@ -165,6 +165,10 @@ path = "examples/app/return_after_run.rs"
 name = "thread_pool_resources"
 path = "examples/app/thread_pool_resources.rs"
 
+[[example]]
+name = "draganddrop"
```

Comment:
Can we keep examples sorted alphabetically?

Reply:
I never realised they were in alphabetical order...